client/templates/includes/gillespyResults.pug

```diff
@@ -18,21 +18,21 @@ div#workflow-results.card.card-body
 
         tbody
           tr
-            td: div(data-hook="title")
-            td: div(data-hook="xaxis")
-            td: div(data-hook="yaxis")
+            td: div(id="title", data-hook="title")
+            td: div(id="xaxis", data-hook="xaxis")
+            td: div(id="yaxis", data-hook="yaxis")
     div.card.card-body
       div
 
         h5.inline Plot Trajectories
-        button.btn.btn-outline-collapse(data-toggle="collapse", data-target="#collapse-trajectories", data-hook="collapse-trajectories") -
+        button.btn.btn-outline-collapse(data-toggle="collapse", data-target="#collapse-trajectories", id="collapse-trajectories", data-hook="collapse-trajectories") -
```

Comment:
The element on line 30 already has this id value I would suggest adding '-btn' to the end of the existing id value.

No element use my suggested id value